### ISSUE TITLE
Same Delay Delays More Than Once

### ISSUE THREAD

**Issue by BryanFRitt** [proginosko/LeechBlockNG#66]
I'm in Firefox-esr 52.8(multi-process). LeechBlockNG does the same delay twice on a page, and then another delay for every new tab, even if it's in the same time period. I made sure I only have one active blocking/delaying set, and the same thing still happens. When it does this it seams to delay the first time right away, but between the first and second delay it shows the page loaded some. I don't have the classic LeachBlock installed.

**Comment by proginosko** [comment]
Thanks for the report. This could be due to site redirection, e.g., if you go to foo.com and it redirects to www.foo.com, LB thinks it's a new site so it shows the delaying page again. Is that the case for the sites you're using? Look at the URL and see whether the domain name changes between the delaying page.

**Comment by BryanFRitt** [comment]
That's seams to be it. e.g. facebook.com redirects to www.facebook.com, If leachblock is set to delay facebook.com, the delay to facebook.com should count towards the delay to www.facebook.com, and towards new tabs.

**Comment by BryanFRitt** [comment]
I didn't notice this until I switched to using the NG version of LeechBlock, and turned on firefox 'multi-process', as well as update from Firefox-esr 52.7.x to 52.8. I did all these at the roughly the same time, so I don't know which of these changes may have caused the issue(s).
I've also done the steps in http://www.proginosko.com/leechblock/faq/start-over/ to fully remove the legacy version LeechBlock.
